isDeskOccupied.py

```diff
@@ -0,0 +1,303 @@
+import requests
+import argparse
+import sys
+import json
+import csv
+import time
+import os
+import tkinter
```

Comment:
make sure that all of the required modules are in the root requirements.txt file